src/compute/src/lib.rs

```diff
@@ -82,12 +82,18 @@ pub struct ComputeNodeOpts {
     /// Endpoint of the connector node
     #[clap(long, default_value = "127.0.0.1:60061")]
     pub connector_source_endpoint: String,
+
+    /// Endpoint of connector sink node
+    #[clap(long, default_value = "127.0.0.1:50051")]
+    pub connector_sink_endpoint: String,
```

Comment:
It is very opaque to implicitly specify a default endpoint address and only report error when we fails to connect to this default endpoint, especially when by default the connector endpoint is not necessarily provided.

We may support storing the endpoint address as None when not explicitly specified by either launch command or env var. We can use clap declaration as `#[clap(long, env = "CONNECTOR_SINK_ENDPOINT")]`. With this declaration, when the endpoint address is not specified in the command line options, it will read from the env var, and if the env var is not set, it then be set None.

You may reference https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.env

Reply:
That makes sense. Do we apply the same to `connector_source_endpoint` when wrapping them in the param struct?